@here/harp-materials/lib/CirclePointsMaterial.ts

```diff
@@ -4,7 +4,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { Color, ShaderMaterial, Uniform } from "three";
+import { Color, IUniform, ShaderMaterial, ShaderMaterialParameters, Uniform } from "three";
```

Comment:
My bad, I did these direct `import from "three"`. Can we change that to the peer dependency as in the other files?